assets/prototype/domain/eventEditor/datesList/dateCard/DateCard.js

```diff
@@ -38,19 +39,30 @@ const idStyle = {
 
 const DateCard = ({ eventId, id }) => {
 	const date = useDateItem({ id });
-	const { updateDate } = useUpdateDateMutation();
-
-	const onTitleConfirm = (name) => {
-		updateDate({
-			variables: { input: { clientMutationId: 'xyz', id, name } }
-		});
-	};
+	const onFieldUpdate = useUpdateDateMutation({ id });
 
 	const startDate = date.start ? new Date(date.start * 1000) : PLUS_ONE_MONTH;
 	// const endDate = date.end ?
 	// 	new Date( date.end * 1000 ) :
 	// 	PLUS_TWO_MONTHS;
 
+	// To be added after proto
+	// const capacityField = (
+	// 	<div>
+	// 		<EditableText
+	// 			type="number"
+	// 			placeholder="Edit capacity..."
+	// 			defaultValue={date.capacity}
+	// 			onCancel={(value) => console.log(value)}
+	// 			onConfirm={(capacity) =>
+	// 				onFieldUpdate({ capacity: Number(capacity) })
+	// 			}
+	// 			minWidth={'320px'}
+	// 			selectAllOnFocus
+	// 		/>
+	// 	</div>
+	// );
+
```

Comment:
@manzoorwanijk as per your reminder, I've commented this out and will add back after proto phase is done.

Reply:
The other logic related to capacity field is not introducing any breaking changes.